contracts/libraries/VaultLifecycleTreasury.sol

```diff
@@ -131,11 +129,13 @@ library VaultLifecycleTreasury {
         );
 
         // get the black scholes premium of the option
-        premium = GnosisAuction.getOTokenPremiumInStables(
-            otokenAddress,
-            optionsPremiumPricer,
-            premiumDiscount
-        );
+        premium = optionsPremiumPricer == address(1)
```

Comment:
Combined `VaultLifecyleTreasuryBare` with existing `VaultLifecyleTreasury`. Since this is the only major change, probably don't need to have two separate files?